app/templates/landing_page.html

```diff
@@ -0,0 +1,97 @@
+{% extends "_base.html" %}
+{% block title %}Home{% endblock %}
+{% block body %}
+
+<br/>
+<section id="identification">
+    <h1>AGS File Utilities Tool and API</h1>
+    <br>
+    <p>This tool and asscoated API allow schema validation, data validation and conversion of your <a href="https://www.ags.org.uk/data-format/">AGS files</a>.</p>
+      <ul>
+        <li>Accepts multiple AGS files. (Hold down <b>Ctrl</b> key to select more than one file)</li>
+        <li>There is a total file size limit of 50MB</li>
+        <li>Tests AGS versions 4.x </li>
+        <li>This is not a submission of your files to BGS</li>
+      </ul>
+</section>
+<section id="validator">
+    <br>
+    <h2>AGS Schema Validator</h2>
+    <h3>Validation is against the official AGS schema standard only. This is not a data validator</h3>
+    <form action="/validatemany/" enctype="multipart/form-data" method="post">
+      <p>Select AGS version (default uses dictionary specified in file, if not present, v4.0.4)</p>
+      <input type="radio" id="4.0.3" name="std_dictionary" value="4.0.3">
```

Comment:
Could the values be
```
v4_0_3
v4_0_4
v4_1
```
?

That would make passing through easier.

Reply:
Sure - done